homeassistant/util/async.py

```diff
@@ -97,6 +98,10 @@ def run_coroutine_threadsafe(coro, loop):
 
     Return a concurrent.futures.Future to access the result.
     """
+    ident = getattr(loop, '_thread_ident', None)
```

Comment:
This is not a very cheap call performance-wise since it walks the hierarchy tree of the loop object searching for the attribute. A cheaper version that ignores class hierarchy would be:

```python
ident = loop.__dict__.get("_thread_ident", None)
```